docker/makefile

```diff
@@ -8,8 +8,24 @@ run: build
 		-v `pwd`/raw:/app/raw:ro \
 		superpixels bash
 
-build:
-	docker build -t superpixels .
+build-buster:
+	docker build --build-arg BASE_IMAGE=debian:buster-20240612-slim   -t superpixels:buster-20240612-slim .
+
+build-bullseye:
+	docker build --build-arg BASE_IMAGE=debian:bullseye-20250520-slim -t superpixels:bullseye-20250520-slim .
+
+build-bookworm:
+	docker build --build-arg BASE_IMAGE=debian:bookworm-20250520-slim -t superpixels:bookworm-20250520-slim .
+
+build-trixie:
+	docker build --build-arg BASE_IMAGE=debian:trixie-20250520-slim   -t superpixels:trixie-20250520-slim .
+
+build: build-buster
```

Comment:
once I figure out what the "latest" I can build is (hopefully `trixie` after the `timer.hpp` fixes), I'll update this.